walkingastro.lic

```diff
@@ -0,0 +1,210 @@
+custom_require.call %w[events]
```

Comment:
💡 Needs a header that links to elanthipedia

Reply:
Righto, will do.